### ISSUE TITLE
Call for maintainers

### ISSUE THREAD

**Issue by realmunk** [nrkno/core-components#494]
I can see that there's quite a few open PR's here, and before we start opening a few more. Is there still people in charge of maintaining this wonderful lib?

**Comment by olanordbryhn** [comment]
We are very pleased to hear about your commitment to, and use of, this library. The development from NRK on this is put on hold for the moment, but will be started again at a later date.

**Comment by realmunk** [comment]
Bump! Har dere noen ny status her, eller må vi begynne å vurdere nye alternativer?

**Comment by EvalenaC** [comment]
Ny status er at vi definitivt kommer til å starte opp videreutviklingen igjen. Akkurat i disse dager rekrutterer vi til en utviklerstilling med fokus på Core components. https://238765.webcruiter.no/Main2/Recruit/Public/4355021099?language=nb&link_source_id=0

New status: We are working on restarting the development for Core components again. Currently we are recruiting for a developer position with focus on Core components. https://238765.webcruiter.no/Main2/Recruit/Public/4355021099?language=nb&link_source_id=0

**Comment by eirikbacker** [comment]
Bump! Meet @skjalgepalg! Our new lead developer for Core-Components 🎉  we will continue development and keep making these components lightweight, insanely accessible and open source ❤️
